packages/spacecat-shared-rum-api-client/test/fixtures/trafficSources.json

```diff
@@ -3,18 +3,36 @@
     "url": "https://www.aem.live/home",
     "total": 2620,
     "earned": 400,
-    "owned": 2220,
-    "paid": 0,
+    "owned": 2120,
+    "paid": 100,
     "sources": [
       {
         "type": "owned:direct",
-        "views": 2220
+        "views": 2120
       },
       {
         "type": "earned:search",
         "views": 400
+      },
+      {
+        "type": "paid:social",
```

Comment:
please use the `{type}:{category}` notation for vendor as well such as:

```
owned:direct
earned:search:google
paid:social:facebook
```

please see https://github.com/adobe/helix-website/blob/main/tools/oversight/test/acquisition.test.js#L17